analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharp/ComparisonKind.cs

```diff
@@ -23,6 +23,6 @@ namespace SonarAnalyzer.SymbolicExecution.SymbolicValues
     public enum ComparisonKind
     {
         Less,
-        LessOrEqual
+        LessOrEqual,
```

Comment:
This `ComparisonKind` is language independent => can you move it to `SymbolicExecution/Common/SymbolicValues`? It was originally in `../SymbolicValues/Binary`

Reply:
I would be also fine with renaming it to `SymbolicComparisonKind` to avoid the collision we have. Its values are reduced from some symbolic execution magic.

Reply:
I applied the renaming.

For the movement:
- [ ] SonarAnalyzer.Common\SymbolicExecution\SymbolicValues
- [ ] SonarAnalyzer.CSharp\SymbolicExecution\Common\SymbolicValues
?

Reply:
This is currently messy because we don't have VB.NET SE (yet). Let's move the file to `SonarAnalyzer.CSharp\SymbolicExecution\Common\SymbolicValues` as there's the rest of the logic to stay focused in this PR.